cmd/vic-machine/create/create.go

```diff
@@ -239,6 +241,12 @@ func (c *Create) Flags() []cli.Flag {
 			Usage:       "Virtual Container Host x509 certificate file",
 			Destination: &c.cert,
 		},
+		cli.StringFlag{
+			Name:        "base-image-size",
```

Comment:
@hmahmood Kind of, not really.  Storage size for "images" implies the size of the image vmdk.  Since we map images to layers on the vmdk, the changing size of the delta vmdk is possible, but not the intent.  This size is effectively the size of the root disk which images are derived from.  Effectively, it's the size of `scratch`.